packages/jest-environment-jsdom-abstract/src/index.ts

```diff
@@ -0,0 +1,191 @@
+/**
```

Comment:
this is a copy paste of the existing env with `jsdom` as a third constructor argument